config/s2i/ostree-boot-image/ostree-boot-image-buildconfig-template.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: ostree-boot-image-builder
+metadata:
+  annotations:
+    description: ostree-boot-image container
+    iconClass: ostree-boot-image
+    tags: instant-app
+  name: ostree-boot-image-builder
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    annotations:
+    labels:
+    name: ostree-boot-image
+  spec: {}
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    name: ostree-boot-image
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ostree-boot-image:latest
+    resources: {}
+    source:
+      contextDir: ${REPO_CONTEXTDIR}
+      git:
+        ref: ${REPO_REF}
+        uri: ${REPO_URL}
+      type: Git
+    strategy:
+      dockerStrategy:
+        env:
+          - name: CONTAINER_NAME
+            value: ostree-boot-image
+      type: Docker
+    triggers:
+    - type: ConfigChange
+parameters:
+- description: Git repository with Dockerfile and slave entrypoint.
+  displayName: Repository URL
+  name: REPO_URL
+  value: https://github.com/CentOS-PaaS-SIG/ci-pipeline.git
+- description: The sub-directory inside the repository.
+  displayName: Context Directory
+  name: REPO_CONTEXTDIR
+  value: config/Dockerfiles/ostree-boot-image
```

Comment:
this does not match up with the actual path in repo...I see '_' in path. I think everything should be '-'

Reply:
where do you see _?